src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -262,7 +268,14 @@ final class RiskProvider: RiskProviding {
 			case .success:
 				completion(.success(()))
 			case .failure(let error):
-				completion(.failure(.failedKeyPackageDownload(error)))
+				if case .noNetworkConnection = error {
+					// we need this delay so the user can see something happened
```

Comment:
Why is this delay not needed for other errors, which might also occur very fast?